src/main/java/model/data/CombinedDataTable.java

```diff
@@ -157,6 +157,20 @@ public CombinedDataRow next() {
 			}
 		};
 	}
+
+	/**
+	 * Return a list that contains all the tables in the combined dataTable.
+	 * @return a list that contains all the tables.
+	 */
+	public LinkedList<DataTable> getTables() {
+		LinkedList<DataTable> res = new LinkedList<>();
```

Comment:
Je kan ook een ArrayList doen en die op het eind reversen, dat is waarschijnlijk sneller.